x-pack/plugins/cases/public/components/all_cases/all_cases_list.tsx

```diff
@@ -126,41 +117,13 @@ export const AllCasesList = React.memo<AllCasesListProps>(
       [queryParams.sortField, queryParams.sortOrder]
     );
 
-    const filterRefetch = useRef<() => void>();
     const tableRef = useRef<EuiBasicTable | null>(null);
-    const [isLoading, handleIsLoading] = useState<boolean>(false);
-
-    const setFilterRefetch = useCallback(
-      (refetchFilter: () => void) => {
-        filterRefetch.current = refetchFilter;
-      },
-      [filterRefetch]
-    );
 
     const deselectCases = useCallback(() => {
       setSelectedCases([]);
       tableRef.current?.setSelection([]);
     }, [setSelectedCases]);
 
-    const refreshCases = useCallback(
```

Comment:
The refresh logic is moved to the child component. They use the `useRefreshCases` hook to refetch.